server/src/main/java/io/crate/metadata/settings/session/SessionSettingRegistry.java

```diff
@@ -159,6 +160,18 @@ public SessionSettingRegistry(Set<SessionSettingProvider> sessionSettingProvider
                      () -> String.valueOf(32),
                      "Shows the maximum number of index keys.",
                      DataTypes.INTEGER))
+            .put(MAX_IDENTIFIER_LENGTH,
+                 new SessionSetting<>(
+                     MAX_IDENTIFIER_LENGTH,
+                     objects -> {},
+                     Function.identity(),
+                     (s, v) -> {
+                         throw new UnsupportedOperationException("\"" + MAX_IDENTIFIER_LENGTH + "\" cannot be changed.");
+                     },
+                     s -> String.valueOf(63),
```

Comment:
Please use directly the constant: https://github.com/crate/crate/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java#L108
(also below)